x-pack/test/security_solution_cypress/cypress/e2e/detection_response/detection_engine/rule_creation/esql_rule_serverless.cy.ts

```diff
@@ -1,38 +0,0 @@
-/*
- * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
- * or more contributor license agreements. Licensed under the Elastic License
- * 2.0; you may not use this file except in compliance with the Elastic License
- * 2.0.
- */
-
-import { getEsqlRule } from '../../../../objects/rule';
-
-import { ESQL_TYPE, NEW_TERMS_TYPE, THRESHOLD_TYPE } from '../../../../screens/create_new_rule';
-
-import { login } from '../../../../tasks/login';
-import { visit } from '../../../../tasks/navigation';
-
-import { CREATE_RULE_URL } from '../../../../urls/navigation';
-import { createRule } from '../../../../tasks/api_calls/rules';
-
-describe('Detection ES|QL rules, creation', { tags: ['@serverless'] }, () => {
```

Comment:
Why is this test being removed?

Reply:
Since ES|QL was enabled on Serverless, it works exactly the same for both ESS and Serverless. So this test is not relevant anymore